oeps/best-practices/oep-0011/decisions/0004-js-ecma.rst

```diff
@@ -0,0 +1,50 @@
+Target the latest standardised JavaScript version
+#################################################
+
+Status
+******
+
+????? it seems like typescript is going to happen
```

Comment:
Again we accept this decision and allow for a future ADR for typescript that can supersede it potentially.